public/components/event_analytics/home/home.tsx

```diff
@@ -72,7 +69,7 @@ const EventAnalyticsHome = (props: IHomeProps) => {
   const history = useHistory();
   const [savedHistories, setSavedHistories] = useState<any[]>([]);
   const [selectedHistories, setSelectedHistories] = useState<any[]>([]);
-  const [isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
+  const [_isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
```

Comment:
If we're not using `_isActionsPopoverOpen`, why are we keeping it? The setter function wouldn't be needed if the getter isn't being accessed anywhere, right?

Reply:
➕

Reply:
Good catch, was used in the event explorer button before redirecting, removed it from both locations.